src/mongo/tree/MongoDatabaseTreeItem.ts

```diff
@@ -28,7 +28,7 @@ export class MongoDatabaseTreeItem implements IAzureParentTreeItem {
 		this._accountConnectionString = accountConnectionString;
 		this._parentId = parentId;
 		const uri: vscode.Uri = vscode.Uri.parse(accountConnectionString);
-		this.connectionString = `${uri.scheme}://${uri.authority}/${this._databaseName}?${uri.query}`;
+		this.connectionString = `${uri.scheme}://${uri.authority}${uri.path}/${this._databaseName}?${uri.query}`;
```

Comment:
Make sure to test non-emulator mongo accounts. This doesn't work in that case:
![screen shot 2018-01-22 at 3 58 53 pm](https://user-images.githubusercontent.com/11282622/35250819-598d6f60-ff8d-11e7-86a2-29f37269c9db.png)

There's an extra slash. You can either remove it after `${uri.path}` and assume the path has it or (even better) do some extra error-checking like this:
```
const path: string = uri.path.endsWith('/') ? uri.path.slice(0, -1) : uri.path;
```

Reply:
I could not repro this error. 
The `uri.path` takes a `"/"` in that case.  I'll try to repro this and then put in the error-checks.